test/DynamicEater.test.ts

```
@@ -0,0 +1,95 @@
+
+		// 	const reward = await merge.getTotalWeWe(100n);
+		// 	expect(reward).to.be.eq(120000116666666666n);
+
+		// 	// // Merge 100 vult to wewe
+		// 	// await merge.connect(otherAccount).merge(100n);
+		// 	// const vested = await merge.vestings(otherAccount.address);
+
+		// 	// expect(vested.amount).to.be.eq(50n);
+		// });
+
+		it.only("Should decrease deposit rates with large numbers", async () => {
```

Biome

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)